### PR TITLE
NETOBSERV-1398: fix loki url when same namespace

### PR DESCRIPTION
**Pull request by jotak**
## Description

Host like "loki-gateway-http.svc" seem invalid

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [x] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [x] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: pkg/helper/loki_config.go [hunk 1]
```diff
@@ -31,15 +31,15 @@ func NewLokiConfig(spec *flowslatest.FlowCollectorLoki) LokiConfig {
...
-			dotNamespace = "." + spec.LokiStack.Namespace
+			ns = spec.LokiStack.Namespace
 		}
-		gatewayURL := fmt.Sprintf("https://%s-gateway-http%s.svc:8080/api/logs/v1/network/", spec.LokiStack.Name, dotNamespace)
+		gatewayURL := fmt.Sprintf("https://%s-gateway-http.%s.svc:8080/api/logs/v1/network/", spec.LokiStack.Name, ns)
 		loki.LokiManualParams = flowslatest.LokiManualParams{
 			QuerierURL:  gatewayURL,
 			IngesterURL: gatewayURL,
-			StatusURL:   fmt.Sprintf("https://%s-query-frontend-http%s.svc:3100/", spec.LokiStack.Name, dotNamespace),
+			StatusURL:   fmt.Sprintf("https://%s-query-frontend-http.%s.svc:3100/", spec.LokiStack.Name, ns),
```
Comment (msherif1234) [inline pkg/helper/loki_config.go:42]: when ns is `""` there is no need to have `.` in the url name right ?

Comment (jotak) [reply pkg/helper/loki_config.go:42]: When namespace is "", the URL won't be correct, no matter if there's the `.` or not, this is precisely the issue that this PR is fixing. So here, an empty namespace will still result in an invalid URL but it is ultimately replaced with the netobserv's namespace which fixes it all.

**Pull request merged**